src/Controller/OpenyMemberships.php

```diff
@@ -340,16 +340,24 @@ public function getProductsInBranch($branch, $data) {
           }
         }
         if (!$filter_product) {
+          $branch_array = NULL;
+          $branch_nodes = $product->field_product_branch->referencedEntities();
+          foreach ($branch_nodes as $branch_node) {
+            if($branch_node->id() !== $branch) {
+              continue;
+            }
+            $branch_array = [
+              'uuid' => $branch_node->uuid(),
+              'id' => $branch_node->id(),
+              'title' => $branch_node->label(),
+            ];
+          }
```

Comment:
I think we should replace the logic from using `continue` with `break`.

The use of break is more efficient and makes the code clearer compared to continue. By breaking the loop immediately after finding the desired branch, we reduce unnecessary iterations, which is especially beneficial with large datasets. This approach also improves code readability by clearly indicating the loop's termination condition.

E.G.
```
foreach ($branch_nodes as $branch_node) {
  if ($branch_node->id() === $branch) {
    $branch_array = [
      'uuid' => $branch_node->uuid(),
      'id' => $branch_node->id(),
      'title' => $branch_node->label(),
    ];
    break;
  }
}
```